cmd/backend/main.go

```diff
@@ -62,19 +63,31 @@ func main() {
 
 	// can have frontend parameters come go straight into simEnv
 	foodOnPlatform := 100.0
-	numOfAgents := []int{2, 2, 2, 2, 2}
+	numOfAgents := []int{2, 2, 2, 2, 2} //agent1, agent2, team3, team6, randomAgent
 	agentHP := 100
 	agentsPerFloor := 1 //more than one not currently supported
 	numberOfFloors := simulation.Sum(numOfAgents) / agentsPerFloor
 	ticksPerFloor := 10
 
 	ticksPerDay := numberOfFloors * ticksPerFloor
-	simDays := 3
+	simDays := 8
 	reshuffleDays := 1
 	dayInfo := day.NewDayInfo(ticksPerFloor, ticksPerDay, simDays, reshuffleDays)
 
+	// define heath parameters
+	strongLevel := 55
+	healthyLevel := 25
+	weakLevel := 5 // larger than maxCriticalDay
+	maxDayCritical := 3
+	foodReqStrong := 20.0
+	foodReqHealthy := 15.0
+	foodReqWeak := 10.0
+	foodReqCToW := 2.0
```

Comment:
Also what are opinions on putting these straight into the NewHealth struct?

Reply:
Indeed, it actually changes depending on where you are in the bracket. These values are only valid if your hp is at the lower border of a given HP level. It can actually be misleading, maybe I should change the name. It is basically the parameter "tau" of the equation

Reply:
We could put them strain into the NewHealth struct yes. If the others agree I can change that

Reply:
Ah understood, yeah I think straight in is better. They don't seem like they'll be variable once finalised. Opinions welcome @JaafarRammal @woonmoon @jzzheng22

Reply:
I changed the them for `tauStrong`, `tauHealthy` and `tauWeak`

Reply:
@udai-arneja good point. I think we can go with this for now, and if the function needs to be changed for health / experimented with we can add even more parametrisation